protected/humhub/libs/DynamicConfig.php

```diff
@@ -102,8 +102,6 @@ public static function rewrite()
         if (!empty($defaultTimeZone)) {
             $config['timeZone'] = $defaultTimeZone;
             $config['components']['formatter']['defaultTimeZone'] = $defaultTimeZone;
```

Comment:
We should also unset() the `formatterApp` for existing dynamic config files.

e.g. https://github.com/humhub/humhub/blob/6278ca0f28f34017bf0e541ee4fb577373770cd5/protected/humhub/libs/DynamicConfig.php#L168

Reply:
@luke- Ok, done https://github.com/humhub/humhub/pull/5048/commits/60beb5a0d53d8787e8385132b31ada9ceeb11919.